extensions/vscode/src/api/resources/Packages.ts

```diff
@@ -43,15 +43,23 @@ export class Packages {
   // 200 - success
   // 400 - bad request
   // 500 - internal server error
-  createPythonRequirementsFile(python?: string, saveName?: string) {
-    return this.client.post<void>("packages/python/scan", { python, saveName });
+  createPythonRequirementsFile(
+    python?: string,
+    saveName?: string,
+    params?: { dir?: string },
+  ) {
+    return this.client.post<void>(
+      "packages/python/scan",
+      { python, saveName },
```

Comment:
Encode the save name as well here?

Reply:
This is in the body so should be good. We just need to encode URI elements.